### PR TITLE
Experiment for only recurring donations on covid19response form

### PR DESCRIPTION
**Pull request by ryan-ludwig**
GROW-104

Add a prop to show one-time donation in lightbox. 
Add ID prop to <multi-amount-selector so we can have multiple instances on one page.

### CODE REVIEW

File: src/pages/LandingPages/MGCovid19/MGCovidFAQ.vue [hunk 1]
```diff
@@ -3,72 +3,98 @@
...
-				<h4>{{ question.title }}</h4>
-				<p>{{ question.content }}</p>
+			<li>
+				<h4>Are my contributions loans or donations?</h4>
+				<p>All contributions to the Global COVID-19 response fund are loans.</p>
+			</li>
+			<li>
+				<h4>Can I just contribute to the response fund once, rather than monthly?</h4>
+				<p>
+					Yes. If you choose “One-time support”, the selected amount will only be deposited once.
```
Comment (ryan-ludwig) [inline src/pages/LandingPages/MGCovid19/MGCovidFAQ.vue:15]: Expecting a text change from marketing here

File: src/pages/LandingPages/MGCovid19/MGCovidFAQ.vue [hunk 2]
```diff
@@ -3,72 +3,98 @@
...
 		};
-		/* eslint-enable max-len */
 	},
+	methods: {
+		showLightbox() {
+			this.isLightboxVisible = true;
+			this.$kvTrackEvent(
+				'Monthly Good',
+				'EXP-GROW-104-May2020',
+				'onetime-lightbox-opened' // TODO: Verify this is the event we want
```
Comment (ryan-ludwig) [inline src/pages/LandingPages/MGCovid19/MGCovidFAQ.vue:90]: Awaiting verification here

File: src/components/Kv/KvLightbox.vue [hunk 3]
```diff
@@ -156,7 +159,7 @@ export default {
 
 			.lightbox-columns {
 				position: relative;
-				width: 100%;
```
Comment (ryan-ludwig) [inline src/components/Kv/KvLightbox.vue:159]: This allows the lightbox to be smaller than 100% if the content is narrow.

Comment (emuvente) [reply src/components/Kv/KvLightbox.vue:159]: We might want to double-check if anything was relying on this for centering the content within the lightbox

Comment (ryan-ludwig) [reply src/components/Kv/KvLightbox.vue:159]: Open all the lightboxes! I'll take a look through this today :)

Comment (ryan-ludwig) [reply src/components/Kv/KvLightbox.vue]: Going to break lightbox changes into a different PR.

**Pull request merged**